can_interface: can0
drawer_btn: 1
node_btn: 2
